.gitignore

```diff
@@ -1,5 +1,6 @@
-package-lock.json
-yarn.lock
 node_modules/
-npm-debug.log
 dist/
+
+yarn.lock
```

Comment:
I think we should commit lock file. Maybe we can discuss of that.
If we keep the lock file, we need to decide about either commit `npm` lock file or the `yarn` one.
In my opinion, we should commit the `npm` one.

Reply:
I can do npm for now. We can discuss later :)

Reply:
I agree to keep the npm since npm is the main tool even if yarn has its advantages. What do you think?

Reply:
going to go with npm and package-lock